File: CarrinhoFlutter/lib/entities/restinpeace/ecartitem.dart
//region imports
//author Bruno Tezine
import 'package:json_annotation/json_annotation.dart';
import 'package:decimal/decimal.dart';
part 'ecartitem.g.dart';
//endregion

@JsonSerializable(nullable: true)
class ECartItem{
	String id;
	String name;
	double amount;
	double unitPrice;
	double totalPrice;

	ECartItem({
		this.id,
		this.name,
		this.amount,
		this.unitPrice,
		this.totalPrice
});

	factory ECartItem.fromJson(Map<String, dynamic> json) => _$ECartItemFromJson(json);
	Map<String, dynamic> toJson() => _$ECartItemToJson(this);
}

File: Portal/Entities/ECartItem.cs
using CarrinhoVirtual.Codes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CarrinhoVirtual.Entities {
    [RestInPeaceEntity]
    public class ECartItem {
        public string Id { get; set; }
        public string Name { get; set; }
        public Decimal Amount { get; set; }
        public Decimal UnitPrice { get; set; }
        public Decimal TotalPrice { get; set; }
    }
}
